test-suite/shared/src/test/scala/org/scalajs/testsuite/niocharset/CharsetTest.scala

```diff
@@ -67,4 +70,24 @@ class CharsetTest {
 
     assertFalse(Charset.isSupported("this-charset-does-not-exist"))
   }
+
+  @Test def aliases(): Unit = {
+    assertEquals(Charset.forName("UTF-8").aliases(),
+      javaSet("UTF8", "unicode-1-1-utf-8"))
```

Comment:
This is in fact also the continuation of the line starting with `assertEquals`, so this should be indented 4 spaces.